content/millennium/r4/clinical/care-provision/nutrition-order.md

```diff
@@ -53,13 +53,13 @@ Search for NutritionOrders that meet supplied query parameters:
 
 ### Parameters
 
- Name              | Required?                      | Type          | Description
+ Name              | Required                       | Type          | Description
 -------------------|--------------------------------|---------------|-----------------------------------------------------------------------
- `_id`             | This or `patient`              | [`token`]     | The logical resource id associated with the resource.
- `patient`         | This or `_id` or `subject`     | [`reference`] | Who the nutrition order is for. Example: `12345`
- `status`          | No                             | [`token`]     | The status of the nutrition order. Example: `active`
- `_lastUpdated`    | No                             | [`date`]      | An explicit or implied date-time range within which the most recent clinically relevant update was made to the nutrition order. Must include a time, and must be prefixed by ‘ge’ or ‘le’. Example: `ge2014-05-19T20:54:02.000Z`
-[`_count`]         | No                             | [`number`]    | The maximum number of nutrition orders to include in a page.
+ `_id`             | Conditionally                  | [`token`]     | The logical resource ID associated with the resource. This parameter is required if `subject` is not used. Example: `11111111`
+ `patient`         | Conditionally                  | [`reference`] | Who the Nutrition Order is for. This parameter is required if the `_id` parameter is not used. Example: `12345`
```

Comment:
only need to capitalize this if we use the resource name (NutritionOrder). this could probably just be left as "order" though, since the whole document is only for nutrition type orders